WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Package and Rate Selection/WooShippingAddPackageViewModel.swift

```diff
@@ -140,10 +140,10 @@ final class WooShippingAddPackageViewModel: ObservableObject {
         }
         let customSavedPackages = packages.customPackages.map {
             return $0.toPackageData()
-        }
+        }.sorted { $0.id < $1.id }
```

Comment:
Nice, I saw this today and have a draft PR https://github.com/woocommerce/woocommerce-ios/pull/14740/, I will close my PR but maybe you can take the added test from https://github.com/woocommerce/woocommerce-ios/pull/14740/files#diff-f2995483ff8550f388b3209c6d9e29419e026bc1819860000dfb04b441c47c51? That test checks if after mupltiple loads the packages are in the same order

Reply:
Nice! I added that test here in 924601d.